api/v1alpha1/kafkauser_types.go

```diff
@@ -96,3 +102,10 @@ func (spec *KafkaUserSpec) GetIfCertShouldBeCreated() bool {
 func (spec *KafkaUserSpec) GetAnnotations() map[string]string {
 	return util.CloneMap(spec.Annotations)
 }
+
+func (spec *KafkaUserSpec) GetExpirationSeconds() int32 {
+	if spec.ExpirationSeconds == nil {
+		return 90 * 24 * 3600
```

Comment:
return this value: https://github.com/cert-manager/cert-manager/blob/master/pkg/apis/certmanager/v1/const.go#L26

e.g.

```golang
if spec.ExpirationSeconds == nil {
    return cermanagerv1.DefaultCertificateDuration
}
```

Reply:
Hi @panyuenlau, using time.Duration is little difficult as kube builder validation for minimum 3600 is throwing error, I have modified the code with converting the `cermanagerv1.DefaultCertificateDuration` to int32 and using it. What do you think. Is it fine?